package.json

```diff
@@ -28,7 +28,7 @@
         "typescript": "^3.7.3"
     },
     "scripts": {
-        "build": "rm -rf dist && NODE_ENV=production babel src --out-dir dist --copy-files --ignore __tests__,spec.js,test.js,__snapshots__",
+        "build": "rm -rf dist && NODE_ENV=production BABEL_ENV=cjs babel src --out-dir dist/cjs --copy-files --ignore __tests__,spec.js,test.js,__snapshots__ && NODE_ENV=production BABEL_ENV=esm babel src --out-dir dist/esm --copy-files --ignore __tests__,spec.js,test.js,__snapshots__",
```

Comment:
Not actually related to your PR, but looking at the `--ignore` flag, I don't believe we have a `spec.js` or `test.js` file.  
We also don't have a `__snapshots__` directory.  
And probably it needs to be `/src/__tests__`?  Trying it out locally, I see `__tests__` does end up in the output `dist` directory.

Reply:
Hm, interesting. I just copied it over with one changed argument, as you can see, but I suppose possibly that should be looked into too, heh.